sentences/da/homeassistant_HassGetState.yaml

```
@@ -1,4 +1,34 @@
 language: da
 intents:
   HassGetState:
-    data: []
+    data:
+      - sentences:
+          - hvad er [(status|tilstand) (på|for|i)] <name> [<i_på> <area>]
```

Our expansion rules are translated, so `<name>` is `<navn>` and `<area>` is `<område>` etc.

```suggestion
          - hvad er [(status|tilstand) (på|for|i)] <navn> [<i_på> <område>]
```

---

Removing `i` as well.

```suggestion
          - hvad er [(status|tilstand) (på|for)] <navn> [<i_på> <område>]
```